app/Http/Controllers/API/PostController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\PostRequest;
+use App\Models\Post;
+
+class PostController extends Controller
+{
+    public function __construct()
+    {
+        $this->middleware('auth:api');
+    }
+    public function index()
+    {
+        $post = Post::all();
+        return response()->json($post, 200);
+    }
+
+    public function store(PostRequest $request, Post $post)
```

Comment:
parameter $post anh thấy có vẻ đang truyền dư, tạo post sao lại có biến $post?
```suggestion
    public function store(PostRequest $request)
```

Reply:
dạ